src/NzbDrone.Core/Notifications/Email/EmailService.cs

```diff
@@ -24,27 +24,22 @@ public EmailService(Logger logger)
 
         public void SendEmail(EmailSettings settings, string subject, string body, bool htmlBody = false)
         {
-            var email = new MailMessage();
-            email.From = new MailAddress(settings.From);
+            var email = new MimeMessage();
+            email.From.Add(MailboxAddress.Parse(settings.From));
 
-            settings.To.ToList().ForEach(x => email.To.Add(x));
-            settings.CC.ToList().ForEach(x => email.CC.Add(x));
-            settings.Bcc.ToList().ForEach(x => email.Bcc.Add(x));
+            settings.To.ToList().ForEach(x => email.To.Add(MailboxAddress.Parse(x)));
+            settings.CC.ToList().ForEach(x => email.Cc.Add(MailboxAddress.Parse(x)));
+            settings.Bcc.ToList().ForEach(x => email.Bcc.Add(MailboxAddress.Parse(x)));
 
             email.Subject = subject;
-            email.Body = body;
-            email.IsBodyHtml = htmlBody;
-
-            NetworkCredential credentials = null;
-
-            if (!string.IsNullOrWhiteSpace(settings.Username))
+            email.Body = new TextPart(htmlBody ? "plain" : "html")
             {
-                credentials = new NetworkCredential(settings.Username, settings.Password);
-            }
+                Text = body
+            };
 
             try
             {
-                Send(email, settings.Server, settings.Port, settings.Ssl, credentials);
+                Send(email, settings.Server, settings.Port, settings.Username, settings.Password);
```

Comment:
Thoughts on passing `settings` as one object?

Reply:
probably a good idea given we use almost everything